modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/resource/v1_0/OpenAPIResourceImpl.java

```diff
@@ -53,10 +72,203 @@ public OpenAPIResourceImpl(
 	public Response getOpenAPI(@PathParam("type") String type)
 		throws Exception {
 
-		return _openAPIResource.getOpenAPI(
+		Response openAPI = _openAPIResource.getOpenAPI(
 			_openAPISchemaFilter, _resourceClasses, type, _uriInfo);
+
+		OpenAPI entity = (OpenAPI)openAPI.getEntity();
+
+		List<String> relationshipsEndpoints =
+			_removeRelationshipsOpenAPIEndpoints(
+				entity, _getRelationshipsEndpoints(entity.getPaths()),
+				endpoint -> endpoint.contains(_BATCH));
```

Comment:
Maybe it would be more precise to check if endsWith("/batch")

Reply:
Thinking about why I used `contains` instead of `endsWith` I realized that, with the changes in the endpoint, the rest builder is not generating the batch endpoints, so all of this filtering code is no longer necessary. I will delete It when I update the code.